src/woo_publications/publications/tests/factories.py

```diff
@@ -11,6 +11,15 @@ class PublicationFactory(factory.django.DjangoModelFactory):
     class Meta:  # type: ignore
         model = Publication
 
+    @factory.post_generation
+    def informatie_categorieen(self, create, extracted, **kwargs):
+        if not create:
+            return
+
+        if extracted:
+            for information_category in extracted:
+                self.informatie_categorieen.add(information_category)  # type: ignore reportAttributeAccessIssue
```

Comment:
Additionally, I think you could use `informatie_categorieen.set(extracted)` instead of looping through the items and adding them (and that should also be the most efficient query).

I'm not sure if the `obj` type annotation suggested above would remove the need to `type: ignore`, let's give it a try!